sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -295,6 +296,71 @@ public async Task AnalyzeSentimentWithLanguageTest()
             Assert.AreEqual("Positive", docSentiment.Sentiment.ToString());
         }
 
+        [Test]
+        public async Task AnalyzeSentimentWithOpinionMining()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "The park was clean and pretty. The bathrooms and restaurant were not clean.";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, options: new AnalyzeSentimentOptions() { IncludeOpinionMining = true });
+
+            CheckAnalyzeSentimentProperties(docSentiment, opinionMining:true);
+            Assert.AreEqual("Mixed", docSentiment.Sentiment.ToString());
+        }
+
+        [Test]
+        public async Task AnalyzeSentimentWithOpinionMiningEmpty()
+        {
+            TextAnalyticsClient client = GetClient();
+            string document = "That was the best day of my life!";
+
+            DocumentSentiment docSentiment = await client.AnalyzeSentimentAsync(document, "en", new AnalyzeSentimentOptions() { IncludeOpinionMining = true });
+
+            CheckAnalyzeSentimentProperties(docSentiment);
```

Comment:
not sending `opinionMining:true` as we just want to make sure that `MinedOpinions` is empty